ERCS/erc-7794.md

```diff
@@ -0,0 +1,820 @@
+---
+eip: 7794
+title: Grant Registry
+description: A cross-network registry contract to manage and track grants, enhancing transparency and  interoperability for grant programs.
+author: Guilherme Neves (@0xneves)
+discussions-to: https://ethereum-magicians.org/t/erc-7794-grant-registry/20791
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-22
+---
+
+## Abstract
+
+This proposal introduces a Grant Registry contract designed to standardize the registration, management, and tracking of grants across multiple blockchains. The contract organizes grant data into distinct categories, separating immutable fields such as chain ID and community from mutable fields like milestones and participants. It allows for modular handling of disbursements and the addition of external links for off-chain data. By emitting key events throughout the grant lifecycle, the registry enables external protocols to fetch data efficiently, promoting transparency, interoperability, and better insights into the performance of grant programs.
```

Comment:
You should be specific somewhere early in your document about exactly what a "grant" is. I assume that this is financial/research/etc. type grants, and not a grant of permissions.